### ISSUE TITLE
WiFi Data not loading

### ISSUE THREAD

**Issue by drdreed** [code-for-nashville/nashviva#68]
When toggling wifi it is throwing the following error.

`Uncaught Error: Invalid LatLng object: (a, s)`

**Comment by ebmshenfield** [comment]
Sampling the output of [the source data](https://data.nashville.gov/api/views/4ugp-s85t/rows.json), I'm seeing that the element has changed from 11 to 14. Quick fix, going to push directly to master.

**Comment by ebmshenfield** [comment]
Diff for posterity - just required editing the `index` field in `main.js`.
```
diff --git a/js/main.js b/js/main.js
index 18c9f23..7afff56 100644
--- a/js/main.js
+++ b/js/main.js
@@ -26,7 +26,7 @@ const datasets = [
   {
     icon: 'wifi',
     markers: [],
-    index: 11,
+    index: 14,
     name: 'WiFi',
     url: 'https://data.nashville.gov/api/views/4ugp-s85t/rows.json'
   },
```

Thanks for reporting!

**Comment by mshenfield** [comment]
Didn't realize I was posting as my alter-ego. Closing, fixed in d2ecfc7fe085cdec9e2e055c21ed1535c32136d0
